src/java.base/share/classes/java/time/OffsetTime.java

```diff
@@ -1281,7 +1281,11 @@ public long toEpochSecond(LocalDate date) {
      * use {@link ChronoField#NANO_OF_DAY} as a comparator.
      *
      * @param other  the other time to compare to, not null
-     * @return the comparator value, negative if less, positive if greater
+     * @return the comparator value is less than zero if the {@code other} is before,
```

Comment:
Using before/after here is very wrong. As described in the text above, two instances can be at the same point in time, yet still have a defined sort order.